### ISSUE TITLE
feature/normalize l2

### ISSUE THREAD

**Issue by JasonShin** [JasonShin/kalimdorjs#90]
* **I'm submitting a ...**
[/] feature request

* **Summary**
This ticket aims to implement normalize API that can be used for a wide variety of regression models. 

Equation:
`X_normalized = X - min(X)/(max(X) - min(X))`

_The process of L2 normalization_

Given an example

```
>>> X = [[ 1., -1.,  2.],
...      [ 2.,  0.,  0.],
...      [ 0.,  1., -1.]]
>>> X_normalized = preprocessing.normalize(X, norm='l2')

>>> X_normalized                                      
array([[ 0.40..., -0.40...,  0.81...],
       [ 1.  ...,  0.  ...,  0.  ...],
       [ 0.  ...,  0.70..., -0.70...]])
```

1. Take all the X-axis then `√1^2 + -1^2 + 2^2` = `√6`
2. Divide each number of the X-axis with `√6` and reconstruct the matrix

References:
http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.normalize.html

**Comment by JasonShin** [comment]
I will implement it, probably tomorrow =)

**Comment by JasonShin** [comment]
PR for this open at https://github.com/JasonShin/kalimdorjs/pull/91/

**Comment by JasonShin** [comment]
Merged =)
